content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -0,0 +1,132 @@
+- [Accessing your payment history and invoices](#accessing-your-payment-history-and-invoices)
+- [Frequently asked questions](#frequently-asked-questions)
+
+## Difference between Free, Team and Enterprise plans
+
+A detailed comparison of the different plans can be found in the Studio
+[pricing page](https://studio.iterative.ai/pricing).
+
+## How to sign up for the Team or Enterprise plan
+
+When you sign up, you're on the Free plan by default. Plans are managed at the
+team level, which means that you can
+[change the plan](/doc/studio/change-team-plan-or-size#change-team-plan) in a
+team's settings page. Each team is on its own plan and needs to be managed
+separately.
+
+## How total price is calculated
+
+For Enterprise plans, all invoice amounts are calculated based on the agreement
+between you and Iterative.
+
+For Team plans, invoice amount is calculated based on the number of seats in the
+team. All invoice amounts are calculated for one billing period at a time. A
+billing period is one month long. The price per seat can be found in the Studio
+[pricing page](https://studio.iterative.ai/pricing).
+
+Below is an example calculation:
+
+Suppose you have a team with 5 seats. At a monthly rate of $40 per seat, the
+total payable amount for the team = $40 \* 5 = $200 per month.
+
+## How increasing or decreasing the number of seats affects total cost
+
+If you add members to available seats or remove members without reducing the
+number of seats in your team, there will be no impact in the cost to you.
+
+If you increase the number of seats, your account will be charged a prorated
+amount based on the time remaining in the billing period at the time each seat
+was added. For instance, if your billing period runs from the 15th of last month
+to the 14th of this month, then, if you add 2 seats on the 10th of this month,
+you will be charged for only 5 days for the 2 new seats. In the following
+billing period, the 2 additional seats will be taken into account for the entire
+billing period.
+
+If you reduce the number of seats, you will accordingly receive an account
+credit that you can apply on your next payment.
+
+## Downgrading a plan
+
+At any point during your subscription, you can reduce the number of seats in
+your plan.
+
+To cancel the Team plan and move to the Free plan, reduce the number of seats in
+the team to 1 or 2 and click on Downgrade. When you reduce the number of seats
+in the team, a credit balance will be computed based on the number of days
+remaining in the billing period. This credit balance can be used in future
+payments.
+
+When you downgrade from the Team plan to the Free plan, you will receive a
+refund based on the number of days remaining in the billing period. In case of
+any discrepancies in the refunded amount, please
+[contact us](mailto:support@iterative.ai).
+
+To upgrade to or downgrade from an Enterprise plan, please contact us.
```

Comment:
Shall we have a link for "contact us" with support email?

Reply:
Added.